plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -170,6 +170,11 @@ public static function add_admin_body_classes( $admin_body_class = '' ) {
 			$classes[] = 'woocommerce-embed-page';
 		}
 
+		// Add page ID as a class.
+		if ( $pageId = PageController::get_instance()->get_current_screen_id() ) {
+			$classes[] = $pageId;
+		}
```

Comment:
If this is still necessary, I think it has two minor issues.

1. Variable naming: $pageId uses camelCase instead of the WordPress-standard snake_case
2. Assignment within condition: The assignment is done inside the if condition which can make code harder to read

A better approach would be:


```php
// Add page ID as a class.
$page_id = PageController::get_instance()->get_current_screen_id();
if ( $page_id ) {
    $classes[] = $page_id;
}
```

Reply:
@chihsuan 

> 1. Variable naming: $pageId uses camelCase instead of the WordPress-standard snake_case
> 2. Assignment within condition: The assignment is done inside the if condition which can make code harder to read.

Thank you for catching it 😄 I still make this mistake when I work on both frontend and backend changes. Fixed it in [Fix var case](https://github.com/woocommerce/woocommerce/pull/56187/commits/179f893e591998e679f452ee2bb7a9dcc54b5f8e)